specification/communication/data-plane/Microsoft.CommunicationServicesSms/stable/2021-03-07/communicationservicessms.json

```diff
@@ -0,0 +1,206 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Communication SMS Service",
+    "description": "Azure Communication SMS Service",
+    "contact": {
```

Comment:
We do not need the contact section anymore. Let's remove it

Reply:
Updated